theia-full-docker/Dockerfile

```diff
@@ -180,12 +184,15 @@ ADD $version.package.json ./package.json
 ARG GITHUB_TOKEN
 # using "NODE_OPTIONS=..." to avoid out-of-memory problem in CI
 RUN yarn --cache-folder ./ycache && rm -rf ./ycache && \
-    NODE_OPTIONS="--max_old_space_size=4096" yarn theia build
+    NODE_OPTIONS="--max_old_space_size=4096" yarn theia build ;\
+    yarn theia download:plugins
 EXPOSE 3000
 ENV SHELL=/bin/bash \
     THEIA_DEFAULT_PLUGINS=local-dir:/home/theia/plugins
 
 ENV GOPATH=/home/project
 ENV PATH=$PATH:$GOPATH/bin
+ENV LC_ALL = C.UTF-8
```

Comment:
I think the spaces on either side of `=` are not good - the equal becomes part of the value of the env var:

```
(env) theia@ec807618bd38:/home/project$ echo $LC_ALL
= C.UTF-8

```